client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/TemperatureChart.tsx

```diff
@@ -195,3 +197,19 @@ export const TemperatureChart = () => {
     </Box>
   );
 };
+
+export const TemperatureChart = () => {
+  const { breachConfig, filter, hasData, isLoading, sensors } =
+    useTemperatureChartData();
+  return (
+    <>
+      <Toolbar filter={filter} />
```

Comment:
Ohh, so current filter toolbar is inside the tab selection (rather then above it ?)

Reply:
No - it's in the usual place, above the tab selection. The toolbar is using a Portal to render, so it shows in a location which differs from where it's declared